### PR TITLE
Add ICU port

### PR DESCRIPTION
**Pull request by zekoz**

### CODE REVIEW

**Comment by zekoz** [comment]
Could you make a github fork of https://github.com/unicode-org/icu in emscripten-ports?

File: embuilder.py [hunk 1]
```diff
@@ -102,7 +102,7 @@
       '''
 
 SYSTEM_TASKS = ['compiler-rt', 'libc', 'libc-mt', 'libc-extras', 'emmalloc', 'emmalloc_debug', 'dlmalloc', 'dlmalloc_threadsafe', 'pthreads', 'dlmalloc_debug', 'dlmalloc_threadsafe_debug', 'libcxx', 'libcxx_noexcept', 'libcxxabi', 'html5']
-USER_TASKS = ['al', 'gl', 'binaryen', 'bullet', 'freetype', 'libpng', 'ogg', 'sdl2', 'sdl2-image', 'sdl2-ttf', 'sdl2-net', 'vorbis', 'zlib']
+USER_TASKS = ['icu', 'al', 'gl', 'binaryen', 'bullet', 'freetype', 'libpng', 'ogg', 'sdl2', 'sdl2-image', 'sdl2-ttf', 'sdl2-net', 'vorbis', 'zlib']
```
[the remark this answers is not in the document]
Comment (zekoz) [reply embuilder.py]: Done

File: tools/ports/icu.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Copyright 2015 The Emscripten Authors.  All rights reserved.
```
[the remark this answers is not in the document]
Comment (zekoz) [reply tools/ports/icu.py]: Done

**Comment by zekoz** [comment]
No local changes, compiles as is. Added test case. Added to AUTHORS.

**Comment by kripken** [comment]
Great, thanks @zekoz!

**Pull request merged**